auth/tokenService.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require('jsonwebtoken'); // installed this library
+
+const secrets = require('../config/secrets.js');
+
+module.exports = {
+  generateToken,
+};
+
+function generateToken(user) {
+  const payload = {
+    subject: user.id,
+    username: user.username,
+    roles: ['Student'],
```

Comment:
remember that we hard coded this in the example just to have it, but we would ideally have this in the actual users db and be a property we grab from the user object here as well